access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java

```diff
@@ -515,7 +516,8 @@ public void getChildrenCounts(List<BriefObjectMetadata> resultList, AccessGroupS
 		solrQuery.add("f." + ancestorPathField + ".facet.limit", Integer.toString(Integer.MAX_VALUE));
 		// Sort by value rather than count so that earlier tiers will come first in case the result gets cut off
 		solrQuery.setFacetSort("index");
-
+		LOG.warn("This is the one you're looking for!" + solrQuery.toString());
+		
```

Comment:
I can remove this logging statement, unless you think we should leave something like this in there